crates/runtime/src/dataconnector.rs

```diff
@@ -136,6 +152,15 @@ pub enum DataConnectorError {
         dataset_name: String,
         table_name: String,
     },
+
+    #[snafu(display("Unsupported file format: {format}"))]
+    UnsupportedFileFormat { format: String },
+
+    #[snafu(display("Unsupported compression type for CSV"))]
```

Comment:
If this is specific to CSV, should we rename it to `UnsupportedCsvCompressionType`?

Reply:
Changed